### ISSUE TITLE
Can I run lgsvl simulator at remote server?

### ISSUE THREAD

**Issue by Freehug** [lgsvl/simulator#1288]
I am using a poorly resourced computer so I want to use a powerful computer's server for running lgsvl simulator. But at the terminal, I got this message whenever I launch
$ ./simulator.
$ Set current directory to /home/my_name/lgsvlsimulator-linux64-2020.06
$ Found path: /home/my_name/lgsvlsimulator-linux64-2020.06/simulator

So I have installed libvulkan1 and add the PPA system on this server but I still get the same errors.
my GPU has more than 8Gb memory and also every resource are fine. Can I get the reason why I can't execute the program at the terminal of the server?

**Comment by EricBoiseLGSVL** [comment]
@Freehug yes we have had other users to this.  Please see other issues related to the error logs you are getting.  @zelenkovsky do you have any pointers for this?
